crates/swc_ecma_minifier/tests/fixture/next/33265/static/chunks/d6e1aeb5-38a8d7ae57119c23/output.js

```diff
@@ -8515,7 +8515,7 @@
                 }, silence_1 = function() {
                     if (!silence) {
                         var metaTable;
-                        silence = (metaTable = {
+                        silence = Object.keys(metaTable = {
```

Comment:
Make sense as `Object.keys` (member access) does not modify `metaTable` and the eval order is `Object.keys` => `metaTable = {...}`